website/next.config.js

```diff
@@ -52,6 +52,11 @@ export default withNextra({
       destination: '/en/substreams/README/',
       permanent: false,
     },
+    {
+      source: '/en/arbitrum-faq/',
+      destination: '/en/arbitrum/faq/',
+      permanent: true,
+    },
```

Comment:
```suggestion
    {
      source: '/en/arbitrum-faq/',
      destination: '/en/arbitrum/arbitrum-faq/',
      permanent: true,
    },
```

Reply:
I did change the path cause just to make it more easier. But can revert it
 
![CleanShot 2023-06-15 at 12 16 31](https://github.com/graphprotocol/docs/assets/44710980/8e4cb182-6ac8-42c5-908b-d03fcaea453a)

Reply:
Oh sorry, I totally missed that change. I see that you reverted it. Sorry about that, you're right that it's a bit repetitive as it is... but oh well.